peers.rb

```diff
@@ -24,3 +25,10 @@
   @user = @storage.first_user()
   return "#{@user["firstname"]} (id: #{@user["pkey"]}) is a user"
 end
```

Comment:
This method is still just used for testing purposes, right?

Reply:
Yep!